### PR TITLE
Align class and property descriptions with the new DCAT scope

### LINKED ISSUE #36: Align class and property descriptions with the new DCAT scope
<p>Related issue: <a href="https://github.com/SEMICeu/DCAT-AP/issues/128">SEMICeu/DCAT-AP/issues/128</a></p>

<p>Property and class descriptions are not taking into account that the scope of the new version of DCAT includes other resource types (e.g., data services).</p>

<p>They should be updated accordingly, by replacing catalogue and datasets with the general notion of Resource (<code>dcat:Resource</code>).</p>

<p>If this revision will be implemented, it would be desirable that future versions of DCAT-AP be aligned accordingly.</p>

### PR DESCRIPTION
**Pull request by andrea-perego**
In order to address https://github.com/SEMICeu/GeoDCAT-AP/issues/36 , revisions have been made by listing, when necessary, the different classes / resource types to which a class / property description refers.

The term "resource" was never used in the narrow sense of `dcat:Resource` without a proper qualification, and always complemented with the list of relevant resource types - e.g., "a resource documented in a Catalogue (as a Data Service, Dataset, or another Catalogue)".

Additional revisions include editorial fixes, and harmonisation of how classes are referred to (e.g., when the reference points to a class documented in the specification, it is capitalised).

Revised sections (open the link for a preview):
- [Mandatory classes](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-classes)
- [Recommended classes](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-classes)
- [Optional classes](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-classes)
- Agent properties:
  - [Recommended properties for agent](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-agent)
- Catalogue properties:
  - [Mandatory properties for catalogue](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-catalogue)
  - [Recommended properties for catalogue](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-catalogue)
  - [Optional properties for catalogue](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-catalogue)
- Catalogue record properties:
  - [Mandatory properties for catalogue record](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-catalogue-record)
  - [Recommended properties for catalogue record](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-catalogue-record)
  - [Optional properties for catalogue record](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-catalogue-record)
- Category properties:
  - [Mandatory properties for category](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-category)
- Category scheme properties:
  - [Mandatory properties for category scheme](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-category-scheme)
- Data service properties:
  - [Mandatory properties for data service](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-data-service)
  - [Recommended properties for data service](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-data-service)
  - [Optional properties for data service](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-data-service)
- Dataset properties:
  - [Recommended properties for dataset](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-dataset)
  - [Optional properties for dataset](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-dataset)
- Distribution properties:
  - [Mandatory properties for distribution](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#mandatory-properties-for-distribution)
  - [Optional properties for distribution](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-distribution)
- Kind properties:
  - [Recommended properties for kind](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-kind)
- Location properties:
  - [Recommended properties for location](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-location)
  - [Optional properties for location](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#optional-properties-for-location)
- Period of time properties:
  - [Recommended properties for period of time](https://raw.githack.com/SEMICeu/GeoDCAT-AP/fix-issue-36/drafts/latest/index.html#recommended-properties-for-period-of-time)

### CODE REVIEW

File: drafts/latest/tables/optional-properties-for-data-service.html [hunk 1]
```diff
@@ -337,7 +337,7 @@
 <p><code>xsd:decimal</code></p>
 </td>
 <td>
-<p>This property refers to the minimum spatial separation resolvable in a Data Service, measured in meters.</p>
+<p>This property refers to the minimum spatial separation resolvable in a Data Service, measured in metres.</p>
```
Comment (bertvannuffelen) [inline drafts/latest/tables/optional-properties-for-data-service.html:340]: typo: should stay meters.

File: drafts/latest/tables/optional-properties-for-dataset.html [hunk 2]
```diff
@@ -414,7 +414,7 @@
 <p><code>xsd:decimal</code></p>
 </td>
 <td>
-<p>This property refers to the minimum spatial separation resolvable in a Dataset, measured in meters.</p>
+<p>This property refers to the minimum spatial separation resolvable in a Dataset, measured in metres.</p>
```
Comment (bertvannuffelen) [inline drafts/latest/tables/optional-properties-for-dataset.html:417]: typo: same here meters is ok.

File: drafts/latest/tables/optional-properties-for-distribution.html [hunk 3]
```diff
@@ -305,7 +305,7 @@
 <p><code>xsd:decimal</code></p>
 </td>
 <td>
-<p>This property refers to the minimum spatial separation resolvable in a dataset distribution, measured in meters.</p>
+<p>This property refers to the minimum spatial separation resolvable in a Distribution, measured in metres.</p>
```
Comment (bertvannuffelen) [inline drafts/latest/tables/optional-properties-for-distribution.html:308]: typo also here: meters

**Review by bertvannuffelen — CHANGES_REQUESTED** [review]
most is fine: only a typo has been introduce meters -> metres. Should stay meters.

**Comment by andrea-perego** [comment]
> most is fine: only a typo has been introduce meters -> metres. Should stay meters.

This was meant to use the British English form, consistently with the rest of the document (DCAT2 uses American English instead).

**Comment by bertvannuffelen** [comment]
Hi Andrea,

ok, I did not know this was an American/British english spelling issue.

kr,

Bert

Op wo 2 dec. 2020 om 19:35 schreef Andrea Perego <notifications@github.com>:

**Comment by andrea-perego** [comment]
My fault, @bertvannuffelen . I should have explained it in the comments.

**Comment by bertvannuffelen** [comment]
@andrea-perego ok to merge it.

**Comment by andrea-perego** [comment]
Thanks, @bertvannuffelen .

**Pull request merged**